spec/codebreaker/console_spec.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module Codebreaker
+  RSpec.describe Console do
+    let(:game) { Game.new(Difficulty.find('easy').level) }
```

Comment:
use double to stub Game instance, you need to test only Console logic here and that some Game instance methods was called